builder/config.js

```diff
@@ -14,48 +14,48 @@ const path = require('path');
 
 const generateHtmlFilesForScreenshots = require('./plugins/generate-html-files-for-screenshots');
 const customMarkdownParse = require('./plugins/custom-markdown-parse');
+const parseLevels = require('./plugins/levels-parse');
 const validate = require('./plugins/validate');
 
-
-
 module.exports = (isDevMode = false) => Metalsmith(__dirname)
-	.metadata({
-		site: {
-			base: 'https://30.codelab.fun/',
-			isDevMode
-		}
-	})
-	.source('../snippets')
-	.destination('../static')
-	.use(metalsmithStatic())
-	.use(collections({
-		snippets: {
-			sortBy: 'path',
-			pattern: '*.md',
-		},
-	}))
+  .metadata({
+    site: {
+      base: 'https://30.codelab.fun/',
+      isDevMode
+    }
+  })
+  .source('../snippets')
+  .destination('../static')
+  .use(metalsmithStatic())
+  .use(collections({
+    snippets: {
+      sortBy: 'path',
+      pattern: '*.md',
+    },
+  }))
   .use(copy({
     pattern: 'pages/*.md',
     directory: '.',
     move: true
   }))
-	.use(customMarkdownParse())
-	.use(validate(
-		require('./schemas/snippet').schema
-	))
-	.use(discoverPartials())
-	.use(registerHelpers())
-	.use(markdown({}))
-	.use(permalinks())
-	.use(tags({
-		layout: 'tag.hbs'
-	}))
-	.use(generateHtmlFilesForScreenshots())
-	.use(rename([
-		[path.join('json', 'index.html'), 'data.json'],
-		[path.join('readme', 'index.html'), 'README.md'],
-	]))
-	.use(layouts({
-		engine: handlebars,
-		default: 'snippet.hbs',
-	}));
+  .use(customMarkdownParse())
+  .use(validate(
+    require('./schemas/snippet').schema
+  ))
+  .use(discoverPartials())
+  .use(registerHelpers())
+  .use(markdown({}))
+  .use(permalinks())
+  .use(tags({
+    layout: 'tag.hbs'
+  }))
+  .use(parseLevels('levels'))
```

Comment:
so this create parallel set of tags based on level. correct?

Reply:
yep